nmdc_server/ingest/omics_processing.py

```diff
@@ -36,9 +36,9 @@
 class OmicsProcessing(OmicsProcessingCreate):
     _extract_value = validator("*", pre=True, allow_reuse=True)(extract_value)
 
-    @root_validator(pre=True)
+    @model_validator(mode="before")
```

Comment:
I think changing the mode to `after` allows us to not have to use the type ignore here.

If I am understanding the docs correctly I do think that is what we want here, to validate the rest of the chain first:
https://docs.pydantic.dev/latest/api/functional_validators/#pydantic.functional_validators.AfterValidator

Reply:
I believe that here we're building a dictionary to put in the `annotations` column, so I would expect we want this to run before other validation, since we're passing in a dict.